src/sentry/tasks/integrations/github/pr_comment.py

```diff
@@ -1,8 +1,25 @@
+from typing import List, Tuple
+
 from django.db import connection
 
 from sentry.models import GroupOwnerType
 
 
+def format_comment(issues: List[Tuple[str]]):
+    body = "## Suspect Issues\nThis pull request has been deployed and Sentry has observed the following issues:\n\n"
+
+    for issue in issues:
+        title = issue[0]
+        subtitle = issue[1]
+        formatted_subtitle = subtitle[:47] + "..." if len(subtitle) > 50 else subtitle
+        url = issue[2]
+        body += f"- ‼️ **{title}** `{formatted_subtitle}` [View Issue]({url})\n"
+
+    body += "\n<sub>Did you find this useful? React with a 👍 or 👎</sub>"
```

Comment:
Can you use Unicode representation instead of the emoji's directly? 
`"\U0001F44D"`